docs/migrate/migrate-setup.rst

```diff
@@ -0,0 +1,182 @@
+
+ 
+Import path
+-------------
+
+The import path has changed as follows:
+
+**Legacy**
+
+.. code-block:: python
+
+    from qiskit import IBMQ
+
+**Updated**
+
+.. code-block:: python
+
+    from qiskit_ibm_runtime import QiskitRuntimeService
+
+Save and load accounts
+------------------------------------
+
+Use the updated code to work with accounts.
+
+**Legacy - Save accounts**
+
+.. code-block:: python
+
+    IBMQ.save_account("<IQX_TOKEN>", overwrite=True)
+
+**Updated - Save accounts**
+The new syntax accepts credentials for two different channels. For more information on retrieving account credentials, see the `getting started guide <https://qiskit.org/documentation/partners/qiskit_ibm_runtime/getting_started.html>`_.
+.. code-block:: python
+
+    # IBM cloud channel
+    QiskitRuntimeService.save_account(channel="ibm_cloud", token="<IBM Cloud API key>", instance="<IBM Cloud CRN>", overwrite=True)
+
+    # IBM quantum channel
+    QiskitRuntimeService.save_account(channel="ibm_quantum", token="<IQX_TOKEN>", overwrite=True)
+
+**Legacy - Load accounts**
+
+.. code-block:: python
+
+    IBMQ.load_account()
+
+**Updated - Load accounts**
+
+The new syntax combines the functionality from ``load_account()`` and ``get_provider()`` in one statement. The ``channel`` input parameter is optional. If multiple accounts have been saved in one device and no ``channel`` is provided, the default is ``"ibm_cloud"``.
+
+.. code-block:: python
+
+    # To access saved credentials for the IBM cloud channel
+    service = QiskitRuntimeService(channel="ibm_cloud")
+
+    # To access saved credentials for the IBM quantum channel
+    service = QiskitRuntimeService(channel="ibm_quantum")
+
+
+Channel selection (get a provider)
+------------------------------------------
+
+Use the updated code to select a channel.
+
+**Legacy**
+
+.. code-block:: python
+
+    provider = IBMQ.get_provider(project="my_project", group="my_group", hub="my_hub")
+
+**Updated**
+
+The new syntax combines the functionality from ``load_account()`` and ``get_provider()`` in one statement.
+When using the ``ibm_quantum`` channel, the ``hub``, ``group``, and ``project`` are specified through the new
+``instance`` keyword.
+
+.. code-block:: python
+
+    # To access saved credentials for the IBM cloud channel
+    service = QiskitRuntimeService(channel="ibm_cloud")
+
+    # To access saved credentials for the IBM quantum channel and select an instance
+    service = QiskitRuntimeService(channel="ibm_quantum", instance="my_hub/my_group/my_project")
+
+
+Get the backend
+------------------
+Use the updated code to view backends.
+
+**Legacy**
+
+.. code-block:: python
+
+    backend = provider.get_backend("ibmq_qasm_simulator")
+
+**Updated**
+
+.. code-block:: python
+
+    backend = service.backend("ibmq_qasm_simulator")
```

Comment:
It's worth pointing out your can also specify `instance` in `service.backend()` instead of having to re-initialize a new service.